internal/pkg/term/progress/spinner.go

```diff
@@ -77,7 +77,7 @@ func (s *Spinner) Start(label string) {
 
 // Stop stops the spinner and replaces it with a label.
 func (s *Spinner) Stop(label string) {
-	s.finalMSG(fmt.Sprintln(label))
+	s.finalMSG(fmt.Sprint(label))
```

Comment:
:O nice!!!

1. Do we need to update `spinner_windows.go` with the same changes?
2. Are we certain this doesn't affect other spinner experiences? for example are the spinners when we run `copilot init` working just the same?

Reply:
Windows just works fine :)